compiler/x/codegen/X86SystemLinkage.cpp

```diff
@@ -429,7 +429,7 @@ TR::X86SystemLinkage::createPrologue(TR::Instruction *cursor)
 
    const TR::X86LinkageProperties &properties = getProperties();
 
-   const uint32_t outgoingArgSize = cg()->getLargestOutgoingArgSize();
+   const uint32_t outgoingArgSize = (cg()->isLeafMethod()) ? 0 : cg()->getLargestOutgoingArgSize();
```

Comment:
Reviewing the code that sets `_largestOutgoingArgSize`, I think it's just pedantic caution. Do you want me to remove it?